course/templates/course/course_page.html

```diff
@@ -237,17 +305,29 @@ <h5 class="mb-3">{{review.student.username}}</h5>
           <span class="carousel-control-next-icon" aria-hidden="true"></span>
           <span class="visually-hidden">Next</span>
         </button>
+
+
         </div>
-      </div>
+      {% else %}
+          <p>No reviews available</p>
+      {% endif %}
+
+    </div>
     </div>
   </div>
-
-  {% else %}
-  {% endif %}
-
   </body>
     <script type="text/javascript" src="{% static 'styles/js/mdb.min.js' %}"></script>
     <script type="text/javascript"></script>
-</html>
+    <script>
```

Comment:
As I understand, the best practice is to put scripts [before the end of the body](https://levelup.gitconnected.com/all-about-script-87fea475b976)

Reply:
I will change it

Reply:
> I will change it

Done